schutzbot/Jenkinsfile

```diff
@@ -807,6 +807,10 @@ void run_tests(test_type) {
     }
 
     if (test_type == 'ostree-ng') {
+        environment {
+            QUAY_USERNAME = credentials('quay-username')
+            QUAY_PASSWORD = credentials('quay-password')
+        }
```

Comment:
That's not going to work, it's not a valid syntax. See https://www.jenkins.io/doc/book/pipeline/syntax/#environment and https://github.com/osbuild/osbuild-composer/pull/1400#pullrequestreview-656413994 and the rest of Jenkinsfile.